### ISSUE TITLE
Space Age Integration

### ISSUE THREAD

**Issue by KiwiHawk** [modded-factorio/bobsmods#271]
- [ ] Turbo Belts
- [ ] Stack Inserters
- [ ] New materials
  - Raw
    - Lithium
    - Calcite
    - Holmium ore
    - Tungsten ore
    - Tree seeds
    - Ice
  - Processed
    - Tungsten carbide
    - Tungsten plate
    - Lithium plate
    - Carbon
  - Intermediates
    - Superconductor
    - Supercapacitor
    - Quantum processor
    - Carbon fiber
    - Fusion power cell
  - Fluids
    - Thruster fuel
    - Thruster oxidizer
    - Ammonia
    - Fluroketone
    - Electrolyte
- [ ] Science packs
- [ ] Elevated rails (which buildings can't have rails over them)
- [ ] New machines. Work out which recipes should be craftable in which machine:
  - Foundry
  - Biochamber
  - Electromagnetic plant
  - Cryogenic plant
- [ ] Heating tower vs heat sources
- [ ] Personal battery MK3
- [ ] Mech armor

**Comment by Para0234** [comment]
I know that I'm constantly only mentioning that one, but there's also the modules.

**Comment by KiwiHawk** [comment]
> I know that I'm constantly only mentioning that one, but there's also the modules.

Quality modules? No, they are already done. Added by Quality mod not Space Age

**Comment by Para0234** [comment]
No, I mean modifying the modules to integrate them properly with space age, in order to prevent us having access to the best modules without leaving Nauvis.

**Comment by KiwiHawk** [comment]
Suggestions/concerns from mod page:

https://mods.factorio.com/mod/bobplates/discussion/67ecce3ec79ce8829891b90a

**Comment by 0Navis0** [comment]
You should also check compatibility with popular modded planet:
- Maraxsis
- Cerys
- Muluna
- Moshine
- Corrundum
- Secreta and Frozeta
- Cubium
- Tenebris
- Ligumis
- Castra
- ...

**Comment by lyratcm** [comment]
- [ ] Rocket launch costs - the cost of launcing a rocket seems very high for how many launches are required, heat shielding in particular with a slow production time, higher complexity and needing large numbers of eletrolyers to split pure water needing lots of space and power. my base was needing 120mw of just for the electroylers to produce 0.7 heat shielding a second (10 of which is needed for 1 rocket part)
- [ ] space production of ammo - needs lead which can't be produced in space

**Comment by nicholasgower** [comment]
As the creator of Muluna, I have a few thoughts on how to make it work better with Bob's mods.

Aluminum plates are the main unique resource of Muluna. This works well with vanilla space age and with other modded planets, since Aluminum does not exist on any other planet. If Aluminum plates remain Muluna-exclusive, and all existing building recipes continue to use Aluminum, that would turn Muluna into an important step in producing tier 3 buildings. Low density structures and solar panels would need to be changed so that they are still craftable on Nauvis.

![Image](https://github.com/user-attachments/assets/df4984c1-83fc-4ac2-be1c-20b6e6c9ebf0)

**Comment by nicholasgower** [comment]
Rocket parts also need to be greatly simplified to fit in with how rockets are available earlier in SA than in vanilla.

**Comment by empZealoth** [comment]
Things I've noticed so far/suggestions

- Obviously Bob's needs an earlier, cheaper silo and rockets. I'd like to see a higher tier silo (either with more module slots or innate productivity boost like some of the SA machines)
- expensive electrolysis mode is all wack (not only does it increase the power consumption drastically, it also makes it slower and blocks efficiency modules at all)  and is kind of balance breaking and extremely painful on machine counts
- Some higher circuit tiers should probably depend on Fulgora research?
- Quantum processor should probably be bob-ified?
- SA adds infinite productivity techs and they need some consideration. Especially the circuit one. Currently it just works for a random, mid-tier circuit
- High tier furnaces (the universal electric one and higher) should probably be gated behind Vulcanus and/or foundry-fied?
- Biter science could probably be reworked to use the nest capture mechanic and just get rid of the artifacts alltogether? Could farm them from captured faction nests instead
- Gleba could unlock a better greenhouse? (Or boost greenhouses in some other way, maybe infinite wood productivity for them? :D)
- SA miners need to be bob-ified

Aquilo could either add fusion bots after nuclear bots or nuclear bots could be renamed to fusion bots and be gated behind aquilo?

**Comment by empZealoth** [comment]
Exoskeleton legs 2 are a Chem Science tier reasearch, but they are gated behind T2 modules (gated behind rocket), so effectively they are way higher tier
All Space Age stuff defaults to Processing Units, even though Advanced Processing Units are a thing

**Comment by empZealoth** [comment]
Space Age biolab probably should have higher tiers/module slots for Bobs?
